### ISSUE TITLE
训练loss不下降

### ISSUE THREAD

**Issue by JunZhan2000** [Z-yq/TensorflowASR#45]
您好，我用conformerS模型在AISHELL-1数据集上训练了20个epoch，但是loss一直不下降，准确度为0，似乎没有训练效果。
config用的都是项目默认的配置，只有数据位置改了一下

data.config
> speech_config:
  mel_layer_type: Melspectrogram #Spectrogram/Melspectrogram/leaf
  mel_layer_trainable: False #leaf support train
  add_wav_info: False
  sample_rate: 16000
  frame_ms: 25
  stride_ms: 10
  num_feature_bins: 80
  reduction_factor: 4 #should keep the same with model_config, DS2 : time_reduction_factor *= s[0] for s in 'conv_strides'
  train_list: '/remote-home/jzhan/Datasets/AISHELL-1/train/transcripts.txt'
  eval_list: '/remote-home/jzhan/Datasets/AISHELL-1/dev/transcripts.txt'
  wav_max_duration: 30 # s
  only_chinese: True
  streaming: False
  streaming_bucket: 0.5 #s
  pinyin_map: './asr/configs/dict/pinyin2phone.map'
inp_config:
  vocabulary: './asr/configs/dict/pinyin.txt'
  blank_at_zero: False
  beam_width: 1
tar_config:
  vocabulary: './asr/configs/dict/lm_tokens.txt'
  blank_at_zero: False
  beam_width: 1
augments_config:
  noise:
    active: False
    sample_rate: 16000
    SNR: [0,15]
    noises: './noise'
  masking:
    active: False
    zone: (0.1,0.9)
    mask_ratio: 0.3
    mask_with_noise: False
  pitch:
    active: False
    zone: (0.0,1.0)
    sample_rate: 16000
    factor: (-1,3)
  speed:
    active: False
    factor: (0.9,1.2)
  hz:
    active: False
optimizer_config:
  lr: 0.001
  warmup_steps: 10000
  beta1: 0.9
  beta2: 0.98
  epsilon: 0.000001
running_config:
  batch_size: 32
  train_steps_per_batches: 10
  eval_steps_per_batches: 10
  num_epochs: 20
  outdir: './models'
  log_interval_steps: 300
  eval_interval_steps: 500
  save_interval_steps: 500

conformerS.config
> model_config:
  name: OfflineConformerCTC
  dmodel: 144
  reduction_factor: 4
  num_blocks: 13
  head_size: 36
  num_heads: 4
  kernel_size: 32
  fc_factor: 0.5
  dropout: 0.1
  ctcdecoder_num_blocks: 1
  ctcdecoder_kernel_size: 32
  ctcdecoder_fc_factor: 0.5
  ctcdecoder_dropout: 0.1
  translator_num_blocks: 2
  translator_kernel_size: 32
  translator_fc_factor: 0.5
  translator_dropout: 0.1

训练结果
<img width="478" alt="image" src="https://user-images.githubusercontent.com/43952592/165020263-9aa8240b-b0d1-4e59-b87a-ddc24f63ae6e.png">
<img width="432" alt="image" src="https://user-images.githubusercontent.com/43952592/165020316-e794e450-76b5-4fae-a9fc-334a5653224b.png">
<img width="394" alt="image" src="https://user-images.githubusercontent.com/43952592/165020336-828b540e-7a93-44a9-9b5b-47ef01efd23e.png">

**Comment by JunZhan2000** [comment]
学习率调低可以解决
